ckanext/advancedauth/templates/user/new_user_form.html

```diff
@@ -4,7 +4,7 @@
   {{ form.input("name", id="field-username", label=_("Username"), placeholder=_("username"), value=data.name, error=errors.name, classes=["control-medium"], is_required=True) }}
   {% set fullname_req = h.advancedauth_require_fullname() %}
 
-  {{ form.input("fullname", id="field-fullname", label=_("Full Name"), placeholder=_("First and Last Name"), value=data.fullname, error=errors.fullname, classes=["control-medium"], is_required=fullname_req) }}
+  {{ form.input("fullname", id="field-fullname", label=_("Full Name"), placeholder=_("First and Last Name"), value=data.fullname, error=errors.fullname, classes=["control-medium"], is_required=True) }}
```

Comment:
Why hardcode what was configurable?

Reply:
Sorry about that. Changed it for testing so I wouldn't have to restart the whole container and forgot to update it.